{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "Contract Management",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "10.1.0"
                        },
                        {
                            "status": "affected",
                            "version": "10.1.3"
                        }
                    ]
                },
                {
                    "product": "Emptoris Spend Analysis",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "10.1.0"
                        },
                        {
                            "status": "affected",
                            "version": "10.1.3"
                        }
                    ]
                },
                {
                    "product": "Emptoris Sourcing",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "10.1.0"
                        },
                        {
                            "status": "affected",
                            "version": "10.1.3"
                        }
                    ]
                }
            ],
            "datePublic": "2019-08-13T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "IBM Emptoris Sourcing 10.1.0 through 10.1.3, IBM Contract Management 10.1.0 through 10.1.3, and IBM Emptoris Spend Analysis 10.1.0 through 10.1.3 generates an error message that includes sensitive information that could be used in further attacks against the system. IBM X-Force ID: 164069."
                }
            ],
            "metrics": [
                {
                    "cvssV3_0": {
                        "attackComplexity": "LOW",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "NONE",
                        "baseScore": 4.3,
                        "baseSeverity": "MEDIUM",
                        "confidentialityImpact": "LOW",
                        "exploitCodeMaturity": "UNPROVEN",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "LOW",
                        "remediationLevel": "OFFICIAL_FIX",
                        "reportConfidence": "CONFIRMED",
                        "scope": "UNCHANGED",
                        "temporalScore": 3.8,
                        "temporalSeverity": "LOW",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.0/UI:N/PR:L/I:N/S:U/AV:N/AC:L/C:L/A:N/E:U/RC:C/RL:O",
                        "version": "3.0"
                    }
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "Obtain Information",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2019-08-20T18:25:27",
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://www.ibm.com/support/docview.wss?uid=ibm10880221"
                },
                {
                    "name": "ibm-emptoris-cve20194485-info-disc (164069)",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_XF"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/164069"
                }
            ]
        }
    },
    "cveMetadata": {
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "assignerShortName": "ibm",
        "cveId": "CVE-2019-4485",
        "dateReserved": "2019-01-03T00:00:00",
        "dateUpdated": "2019-08-20T18:25:27",
        "state": "PUBLISHED",
        "datePublished": "2019-08-20T18:25:27.111414Z"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}
